crates/sui-framework-build/src/compiled_package.rs

```diff
@@ -130,26 +130,29 @@ impl BuildConfig {
     /// Given a `path` and a `build_config`, build the package in that path, including its dependencies.
     /// If we are building the Sui framework, we skip the check that the addresses should be 0
     pub fn build(self, path: PathBuf) -> SuiResult<CompiledPackage> {
-        let resolution_graph = if self.print_diags_to_stderr {
-            self.config
-                .resolution_graph_for_package(&path, &mut std::io::stderr())
-                .map_err(|err| SuiError::ModuleBuildFailure {
-                    error: format!("{:?}", err),
-                })?
-        } else {
-            self.config
-                .resolution_graph_for_package(&path, &mut std::io::sink())
-                .map_err(|err| SuiError::ModuleBuildFailure {
-                    error: format!("{:?}", err),
-                })?
-        };
+        let print_diags_to_stderr = self.print_diags_to_stderr;
+        let run_bytecode_verifier = self.run_bytecode_verifier;
```

Comment:
Not in love with doing this, but couldn't find a better way since passing `self` runs into borrow issues.